autopush/base.py

```diff
@@ -39,8 +39,8 @@ def write_error(self, code, **kwargs):
             self.log.failure(
                 format=kwargs.get('format', "Exception"),
                 failure=failure.Failure(*kwargs['exc_info']),
-                **self._client_info)
+                client_info=self._client_info)
```

Comment:
Addressed in later patch. Kibana uses the json_formatting() function, so the client_info data is collapsed up in that function. raven will still get the sub-dict that will contain the full client-info data.